msprime/intervals.py

```diff
@@ -198,9 +308,11 @@ class RecombinationMap:
         the largest possible value, allowing the maximum resolution
         in the recombination process. However, for a finite sites
         model this can be set to smaller values.
+    :param str name: A name to use for this map, and stored in the ``.name``
+        attribute. This will only be used if the map is written out to a file
     """
 
-    def __init__(self, positions, rates, num_loci=None, map_start=0):
+    def __init__(self, positions, rates, num_loci=None, name=None, **kwargs):
```

Comment:
Again, no extra features here, this is legacy code which we just want to preserve for compatability.